internal/jobobject/utilities.go

```diff
@@ -0,0 +1,19 @@
+package jobobject
```

Comment:
I personally think it's fine to leave them in this package if the methods are needed in other places (it used to be the case where only host process containers cared about this), but can we just call this file conversions.go maybe?

Reply:
Can't we just put these in `hcsoci` or something? It seems like they are just about converting from OCI concepts into job object concepts.

Reply:
I was looking at it as the notion of converting to a "processor count" isn't really tied to OCI, but it is what we were using the conversion for. I don't mind if it's in hcsoci

Reply:
I'm trying to push to keep `jobobject` completely unaware of what a container is. I would like it to be something we can put into `go-winio` in the future, so nothing container related should leak into it.

Reply:
Makes sense, I just didn't think a processor count was really container specific and might have other use cases

Reply:
moving these into `hcsoci` is a actually problematic since it creates an import cycle :/

Reply:
Looking at the code, it actually looks like nothing in the shim uses them aside from `jobcontainers`. If we're moving them just to use in test code, maybe it would be better to just duplicate the conversion instead?

Reply:
Good point. I think previously it was also being used for working with my test tool to set cpu limits for pre Vb builds, but since that's gone I can take it out. We'll move it if there's a reason to later.